content/instructor/collections/index.mdx

```diff
@@ -9,6 +9,8 @@ shareImage: 'https://res.cloudinary.com/dg3gyk0gu/image/upload/v1571260698/og-im
 
 If you want a quick way to group together a set of related lessons you can create a collection. A collection is a playlist of videos that can share a common theme and be shared and promoted to your audience.
 
+![](/images/collections-screenshot.png)
```

Comment:
```suggestion
![web browser displaying egghead collection page which includes a title, a description, and a list of videos](/images/collections-screenshot.png)
```

@hiroko when possible I love to add descriptive alt text to images

Reply:
@hiroko I used the "[suggest change](https://haacked.com/archive/2019/06/03/suggested-changes/)" feature for you to review and "commit suggestion" or make additional changes.